popup.html

```diff
@@ -10,8 +10,9 @@
 <body>
     <div class="spinner"></div>
     <div class="linkList">
-        <div>
+        <div class="buttonContainer">
             <button class="saveButton" type="button">Save Tab</button>
```

Comment:
Change this to have an id="saveButton" and update the javascript to target the id not the class